galsim/wcs.py

```diff
@@ -345,6 +397,25 @@ def profileToImage(self, world_profile, image_pos=None, world_pos=None, color=No
         return self.local(image_pos, world_pos, color=color)._profileToImage(
                     world_profile, flux_ratio, PositionD(offset))
 
+    def shearToImage(self, world_shear, image_pos=None, world_pos=None, color=None):
+        """Convert a shear measured in world coordinates to image coordinates
+
+        This is equivalent to ``wcs.toImage(shear, ...)``.
+
+        This reverses the conventions described in `shearToWorld`.
```

Comment:
```suggestion
        This reverses the process described in `shearToWorld`.
```
I think the shear convention is the same and it's the process (of moving between two coordinate systems) that is reversed.

Reply:
Agreed.  That's a better word here.